content/docs/dvclive/live/make_stage_template.md

```diff
@@ -0,0 +1,69 @@
+# Live.make_stage_template()
+
+Writes template for a DVC pipeline [stage], capturing the outputs tracked by
+DVCLive, which you can use to [run] DVCLive experiments with DVC.
+
+```py
+def make_stage_template()
+```
+
+## Usage
+
+```py
+from PIL import Image
+from pathlib import Path
+from dvclive import Live
+
+live = Live(cache_images=True)
+
+# Log an artifact
+Path("model.pt").write_text("big_artifact")
+live.log_artifact("model.pt")
+
+# Log an artifact but don't cache it
+Path("weights.json").write_text("small_artifact")
+live.log_artifact("weights.json", cache=False)
+
+# Log an artifact and copy it to `dvclive/artifacts`
+Path("/tmp/model.pt").write_text("external_artifact")
+live.log_artifact("/tmp/model.pt", copy=True)
+
+# Log an image
+img_pil = Image.new("RGB", (500, 500), (250, 250, 250))
+live.log_image("matplotlib.png", img_pil)
+
+live.make_stage_template()
+```
+
+## Description
+
+Creates `{Live.dir}/stage_template.yaml`, which provides a template for a DVC
+pipeline stage. Copy the contents into a `dvc.yaml` file _outside the DVCLive
```

Comment:
Maybe it is just my bad understanding (I know realize that it should be `Copies` if you were referring to the method)

```suggestion
pipeline stage. You can then copy the contents into a `dvc.yaml` file _outside the DVCLive
```